estimagic/inference/moment_covs.py

```diff
@@ -0,0 +1,58 @@
+"""
+This module provides functions for the calculation of standard errors for anon linear
+GMM estimator. The source of these calculations is
+Bruce E. Hansen - Econometrics (https://www.ssc.wisc.edu/~bhansen/econometrics).
+"""
+import numpy as np
+
+
+def covariance_moments(mom_cond):
+    """
+    Calculate the standard covariance matrix Omega.
+
+    Args:
+        mom_cond (np.array): 2d array matrix of the moment conditions of
+            dimension (nobs, nmoms).
+
+    Returns:
+        cov (np.array): 2d array covariance matrix of the moments (nmoms, nmoms)
+
+    """
+
+    dev = mom_cond - np.mean(mom_cond, axis=0)
+    cov = dev.T @ dev / mom_cond.shape[0]
+    return cov
+
+
+def gmm_cov(mom_cond, mom_cond_jacob, mom_weight):
```

Comment:
`gmm_cov` should be defined at the top of the module. All other functions should be private, i.e. start with and underscore.

Reply:
Reordered.